gdx/src/com/badlogic/gdx/Screen.java

```diff
@@ -24,7 +26,7 @@
  * Note that {@link #dispose()} is not called automatically.
```

Comment:
This comment isn't actually true in this PR's version.

Reply:
It wouldn't be called automatically anyway. Even extending of Disposable interface won't make it automatic, the method has to be called from somewhere. This extension only provides the possibility to work with the Screen as with Disposable item.